inception/inception-api-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/annotation/adapter/SpanAdapter.java

```diff
@@ -77,6 +81,13 @@ public SpanAdapter(LayerSupportRegistry aLayerSupportRegistry,
         }
     }
 
+    @Override
+    public void initialize(AnnotationSchemaService aSchemaService)
```

Comment:
This method is called in order to create default features in the annotation schema when a new layer is being created. This is *not* meant to be used to initialize the adapter itself. It is called only by `LayerDetailForm.actionSave`. You cannot reliably obtain an `annotationSchemaService` in this way.